supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -29,6 +29,27 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 // .Other updates
 // * TODO
 
+[[october-2023]]
+== October 2023
+//
+.Glossary entries
+* xref:ansible-rulebook[Ansible Rulebook] Added glossary entry
+* because: Removed glossary entry because guidance already exists in the _IBM Style_ guidance
+* xref:executable-adj[executable (adjective)] Added glossary entry
+* xref:executable[executable (noun)] Added glossary entry
+* xref:pc[PC]: Updated glossary entry to remove mention of the _IBM Style_ guide
```

Comment:
I don't how nitty gritty we want to get with these updated entries, but this seems like it might not be important enough to mention, since no real guidance changed. wdyt?